cmake/std/atdm/rhel6/tweaks/COMMON_TWEAKS.cmake

```diff
@@ -0,0 +1,2 @@
+# Disable just one Teko sub-unit test that fails on GCC 6.1.0 on RHEL6 (#2652)
+ATDM_SET_ENABLE(Teko_DISABLE_LSCSTABALIZED_TPETRA_ALPAH_INV_D TRUE)
```

Comment:
This isn't a show stopper, but I'm curious why you put this in the cmake system and not in the Teko package? (realize I know next to nothing about the cmake system and what is possible/hard)

Reply:
Yes, this sound reasonable to me. I like the idea of localizing all the particular tests in one place. I hadn't really considered that, given that I am more package focused. But this makes a lot of sense from a testing maintenance perspective.